tst/EnergyPlus/unit/FanCoilUnits.unit.cc

```diff
@@ -2129,8 +2130,8 @@ TEST_F(EnergyPlusFixture, FanCoil_ASHRAE90VariableFan)
     UpdateScheduleValues(*state);
 
     state->dataSize->ZoneEqSizing.allocate(1);
-    state->dataSize->ZoneEqSizing(state->dataSize->CurZoneEqNum).SizingMethod.allocate(DataHVACGlobals::NumOfSizingTypes);
-    state->dataSize->ZoneEqSizing(state->dataSize->CurZoneEqNum).SizingMethod = 0;
+    state->dataSize->ZoneEqSizing(state->dataSize->CurZoneEqNum).SizingMethod.allocate(static_cast<int>(AutoSizingType::Num));
+    state->dataSize->ZoneEqSizing(state->dataSize->CurZoneEqNum).SizingMethod = 0; // TODO: This should not be a numeric literal
```

Comment:
I'm not sure if the value assigned to this Array1D was ever used...I hope not.  When I make this a std::array I won't be able to do this, so we'll see what happens.